arch/csr/tdata2.yaml

```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=../../schemas/csr_schema.json
+
+$schema: "csr_schema.json#"
+kind: csr
+name: tdata2
+long_name: Second Debug/Trace trigger data register
+address: 0x7A2
+priv_mode: M
+length: XLEN
+description: |
+  This register provides access to the trigger selected by tselect. 
+  The reset values listed here apply to every underlying trigger.
+  
+  Trigger-specific data. It is optional if no implemented triggers use it.
+  If the trigger is disabled, then this register can be written with any value 
+  supported by any of the trigger types supported by this trigger.
+  If XLEN is less than DXLEN, writes to this register are sign-extended. 
+  
+  This CSR is read/write.
+
+definedBy: Sdtrig
+fields:
+  DATA:
+    location_rv32: 31-0
+    location_rv64: 63-0
+    type: RW
+    description: Data value
+    reset_value: UNDEFINED_LEGAL
```

Comment:
GitHub is complaining about how this file ends
![image](https://github.com/user-attachments/assets/32611fa2-9564-404e-9da1-f032dbb92db2)
(I can never recall exactly how to fix this. Need a newline?)